docs/blog/Webots-2020-a-release.md

```diff
@@ -0,0 +1,100 @@
+# Version R2020a Released
+
+<p id="publish-data">By Stefania Pedrazzi - 9th December 2019</p>
+
+---
+
+Here we are! It's already time for a new Webots release!
+
+This new Webots version focuses on improving the performance of the scene load and the robustness of the simulation.
+But as usual we also extended our robot and asset library with new environments, worlds and appearances.
+Moreover the [Supervisor API](../reference/supervisor.md) has been extended.
+
+This article only lists some of the major changes.
+Besides these, we are always fixing bugs and improving the quality and robustness of Webots.
+Please refer to the changelog for a comprehensive list of all the changes, found [here](../reference/changelog-r2020.md).
+
+---
+
+## New TIAGo Robots Model
+
+Simulate the TIAGo robot from [PAL Robotics](http://pal-robotics.com/) in Webots.
+
+%figure "Tiago Model in Webots"
+![Tiago Model in Webots](images/tiago.jpg)
+%end
+
+Several models of the robot have been created for Webots:
+  - [TIAGo Base](../guide/tiago-base.md)
+  - [TIAGo Iron](../guide/tiago-iron.md)
+  - [TIAGo Steel](../guide/tiago-steel.md)
+  - [TIAGo Titanium](../guide/tiago-titanium.md)
+  - [TIAGo++](../guide/tiagopp.md)
+
+---
+
+## Extended Supervisor API
+
+We are always collecting feedbacks form users, and in particular recently we received many requests about adding new functionalities to the [Supervisor API](../reference/supervisor.md).
+
+So for this release we extended the [Supervisor API](../reference/supervisor.md) features to be able to:
+  - Add [force](../reference/supervisor.md#wb_supervisor_node_add_force) or [torque](../reference/supervisor.md#wb_supervisor_node_add_torque) to a [Solid](../reference/solid.md) node.
+  - [Import](../reference/supervisor.md#wb_supervisor_field_import_sf_node) and [remove](../reference/supervisor.md#wb_supervisor_field_remove_sf) nodes in SFNode fields.
+  - [Reset](../reference/supervisor.md#wb_supervisor_simulation_reset) the simulation without restarting the controllers.
+
+We also improved the [rest of the physic](../reference/supervisor.md#wb_supervisor_simulation_reset_physics) to avoid explosion of dynamic articulated objects.
```

Comment:
```suggestion
We also improved the [reset of the physics](../reference/supervisor.md#wb_supervisor_simulation_reset_physics) to avoid dynamics explosions with articulated objects.
```